guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -5,19 +5,19 @@
 Use this procedure to enable the repositories that are required to install {ProductName}. Choose from the available list which operating system and version you are installing on:
 
 ifdef::foreman-el,katello[]
-* xref:#storage-centos-7[CentOS 7]
-* xref:#storage-centos-8[CentOS 8]
+* xref:#repositories-centos-7[CentOS 7]
+* xref:#repositories-centos-8[CentOS 8]
 endif::[]
 ifdef::foreman-el,katello,satellite[]
-* xref:#storage-rhel-7[Red Hat Enterprise Linux 7]
+* xref:#repositories-rhel-7[Red Hat Enterprise Linux 7]
 endif::[]
 ifdef::foreman-deb[]
-* xref:#storage-debian-10[Debian 10 (Buster)]
-* xref:#storage-ubuntu-1804[Ubuntu 18.04 (Bionic)]
```

Comment:
Hi @adahms 
the problem is that I cannot see this in the 2.3 branch and the downstream is supposed to build off the 2.3 branch rather than our master.

Reply:
Thanks @melcorr - I'm with you now.

If it's of any use, I agree that these shouldn't get cherry-picked into the 2.3 branch, but they did come up as issues to me in the master branch here. Not sure if you're open to merging them there.

I'll be AFK for a bit, but will come back and set up something separate for the 2.3 branch.

Apologies for the confusion.

Reply:
@adahms we have had little to no collaboration with CCS in the past while. 
No one updated the downstream satellite sync branch for 6.9
@spetrosi has just done this. 
Whenever you get a chance, can you check pantheon? I think the content management issue needs to be in 6.9 but the rest only in Master for now.

Reply:
Thanks, @melcorr - changes came through, resolved everything except for the Content Management Guide, which I'm looking at now. Seems to be closely related to - but different from - the change for that same guide here.

I'll raise a separate PR for that now if needed, and you're welcome to accept or discard the changes suggested in this PR for master.